mod/log/pkg/phuslu/config.go

```diff
@@ -20,18 +20,22 @@
 
 package phuslu
 
-import "time"
-
 // Config is a structure that defines the configuration for the logger.
 type Config struct {
 	// TimeFormat is a string that defines the format of the time in
 	// the logger.
-	TimeFormat string
+	TimeFormat string `mapstructure:"time-format"`
+	// Verbose is the level of verbose for the logger.
+	Verbose bool `mapstructure:"verbose"`
```

Comment:
@ocnc2 convert this to the log level

"info", "trace", "debug" etc.

Reply:
yep sounds good